doc/js/scripts/vendor/svg-pan-zoom.min.js

```diff
@@ -0,0 +1,3 @@
+// svg-pan-zoom v3.6.2
```

Comment:
Can we include the license to the top of this file?

https://github.com/bumbu/svg-pan-zoom/blob/master/LICENSE